packages/grafana-ui/src/components/DataSourceSettings/AlertingSettings.tsx

```diff
@@ -27,6 +48,25 @@ export function AlertingSettings<T extends { manageAlerts?: boolean }>({
             />
           </div>
         </div>
+        {(options.type === 'loki' || options.type === 'prometheus') && (
```

Comment:
not sure that htis check is needed, <AlertingSettings/> are only added to Loki & Prometehus datasource configs anyway